### PR TITLE
Fix some errors in _index.html

### PR DESCRIPTION
**Pull request by TFKls**
I've tried to fix some language mistakes I've found on the home page of CPEditor.
I may have missed some errors, and It's also possible that I made some other mistakes in the process. 
In any of the above cases, I would be glad someone pointed them out and/or corrected them.
For now, I've only changed _index.html
I also changed some parts due to stylistic preference.

Have a nice day, and thanks for creating this editor. :)

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -41,15 +41,15 @@ <h2>The Missing Editor <br />for Competitive Programmers</h2>
 <div class="work_area p_120">
 	<div class="container">
 		<div class="main_title">
-			<h2>How It work for you</h2>
-			<p>CP Editor eliminates all the repetitive and boring tasks that you have to do during a contest, it automates all the automatable task allowing you to only focus on your code.</p>
+			<h2>How it works for you</h2>
+			<p>CP Editor eliminates all the repetitive and boring tasks present during algorithmic contests. It automates all that's reasonably possible and allows you to purely focus on your code.</p>
```
Comment (coder3101) [inline content/en/_index.html:45]: > CP Editor eliminates all the repetitive and boring tasks ~present~ during algorithmic...

File: content/en/_index.html [hunk 2]
```diff
@@ -96,7 +96,7 @@ <h2>How It work for you</h2>
 							<div class="left_side_text">
 								<h3>Run any code<br />At any time</h3>
 								<h6>We know that you want to test something without a temp file</h6>
-								<p>You can run codes in one click without saving them by yourself, and the hot exit feature makes it easy to restore from your last session. But if you have chosen where to save, the executable files will be there, too.</p>
+								<p>You can run your programs in one click without saving them by hand and the hot exit feature makes it easy to restore your code from the last session. If you have however chosen where to save your source code, the executable files will be there as well.</p>
```
Comment (coder3101) [inline content/en/_index.html:99]: Actually CP Editor runs your code. Running your program as in running already compiled  programs. I think code is better than program.

> ... without saving them ~by hand~ and the hot exit ....

> If you have however chosen

Could it be rephrased?

File: content/en/_index.html [hunk 3]
```diff
@@ -113,7 +113,7 @@ <h6>We know that you want to test something without a temp file</h6>
 							<div class="left_side_text">
 								<h3>Dark themes? <br />We've got you covered</h3>
 								<h6>Our lives are colorful, why not our coding editor</h6>
-								<p>We have added five different themes for you to choose from Light, Drakula, Monkai, Solarised and Solarised Dark. If that does not satisfy you, ask us on our community and we will provide you more.</p>
+								<p>We have added five different themes for you to choose from Light, Drakula, Monkai, Solarized and Solarized Dark. If that does not satisfy you, ask us on our community and we will provide you more.</p>
```
Comment (coder3101) [inline content/en/_index.html:116]: > Solarized **Light**


> **Monokai**

We've changed the names.

File: content/en/_index.html [hunk 4]
```diff
@@ -181,7 +181,7 @@ <h2>Unique Screens that work perfectly</h2>
 	<div class="container">
 		<div class="main_title">
 			<h2>Feedback from Community</h2>
-			<p>Our Community support us and keep us alive, and we love to hear their feedback<br>See what other Competitive Programmers have to say about our editor</p>
+			<p>Our Community support us and keep us alive. We would love to hear their feedback<br>Come and see what other Competitive Programmers have to say about our editor</p>
```
Comment (coder3101) [inline content/en/_index.html:184]: > love to hear **your** feedbacks

> ~Come and~ **See**

Some minor improvements.

File: content/en/_index.html [hunk 5]
```diff
@@ -264,7 +264,7 @@ <h4>Nitish Kumar Singh</h4>
 					<div class="left_side_text">
 						<h3>Full support for<br />C/C++, Java and Python</h3>
 						<h6>Use the language you like</h6>
-						<p>We officially support three languages in the editor. You can set the compile commands and run arguments. It really gets exciting when we give you the opportunity to use different languages in different tabs with one-click options.</p>
+						<p>We officially support three languages in the editor. You can set the compilation commands and flags independently. It really gets exciting when you have the opportunity to use different languages in different tabs with one-click options.</p>
```
Comment (coder3101) [inline content/en/_index.html:267]: Compilation flags are counted in Compilation command. It's actually that runtime arguments are seldom used in CP.

File: content/en/_index.html [hunk 6]
```diff
@@ -21,7 +21,7 @@
 				<div class="col-lg-5">
 					<div class="banner_content">
 						<h2>The Missing Editor <br />for Competitive Programmers</h2>
-						<p>CP Editor is specially designed for competitive programming. It helps you focus on your algorithm and automates the compilation, execution and testing. It even fetches test cases for you from different platforms and submits solutions to Codeforces!</p>
+						<p>CP Editor is specifically designed for competitive programming. It helps you focus on your algorithms, while it takes care of compilation, execution and testing. It even fetches test cases for you from different platforms and submits solutions to Codeforces!</p>
```
Comment (ouuan) [inline content/en/_index.html:24]: I think "specially" is better. I think "specifically" is like "specially" + "only", but you can use CP Editor in other places, like running any code without saving it to a file.

Can you explain why you think "specifically" is better?

---

Why is "takes care of" better? I think "automates" makes it easier to understand what it is exactly doing. I also think "and" is better than "while" here.

File: content/en/_index.html [hunk 7]
```diff
@@ -129,8 +129,8 @@ <h6>Our lives are colorful, why not our coding editor</h6>
 						<div class="col-lg-6">
 							<div class="left_side_text">
 								<h3>Errors and Warnings?<br />We lint them while you are coding</h3>
-								<h6>Save your precious contest time by code linting</h6>
-								<p>Using Language Server, we provide real-time linting for errors and warnings in your code, just like Intellisense in VSCode and Atom. You can see squiggles under the warnings and hover over them to get tooltips.</p>
+								<h6>Save your precious contest time by using code linting</h6>
```
Comment (ouuan) [inline content/en/_index.html:132]: Why "using"? The subject of "using" is the user, so with "using" the sentence means "the user saves their time", but I think without "using", there's nothing wrong, and it means "the editor saves the user's time".

Comment (TFKls) [reply content/en/_index.html:132]: As far as I see it, without the specific "using" the code linting could be interpreted as referring to the user, which implies the user does manual code linting.
I could see "with code linting" work here, but "by" feels out of place here, as (the way I see it) there isn't really an implied editor it that sentence.

File: content/en/_index.html (hunk 6 above)
Comment (TFKls) [reply content/en/_index.html:24]: Yeah, I did kind of change the context with "specially" so that's my bad.
I changed "automates" because I felt it was overused a bit. (The old version had "automates all the automatable task" which was really unnatural I felt) It can also be reverted.
"And" and "While" both work, changing the context just slightly.

File: content/en/_index.html (hunk 3 above)
Comment (TFKls) [reply content/en/_index.html:116]: Yeah, missed that. My bad.

File: content/en/_index.html (hunk 1 above)
Comment (TFKls) [reply content/en/_index.html:45]: Both work, although I thought "present" kept more of the original meaning.

File: content/en/_index.html (hunk 2 above)
Comment (TFKls) [reply content/en/_index.html:99]: About the rephrasing, I think it would be the best to delete the whole sentence, since it does not add much and saving code in a code editor is not an unique feature.

**Comment by TFKls** [comment]
I've added most of the review suggestions as best as I could think of.
On some reviews, I've left additional information detailing my choices.

File: content/en/_index.html (hunk 6 above)
Comment (ouuan) [reply content/en/_index.html]: I still think "specially designed" is better. "with ... in mind" sounds like competitive programming is only one of the things we've considered, but we are specially considering competitive programming.

File: content/en/_index.html (hunk 2 above)
Comment (ouuan) [reply content/en/_index.html]: >  it does not add much and saving code in a code editor is not an unique feature

The feature here is not saving the code. It's "run codes without saving the codes to a file". I think most IDEs require saving the code before compiling it?

Comment (ouuan) [reply content/en/_index.html]: It seems that you misunderstood this, can you rephrase it to make it clearer?

File: content/en/_index.html (hunk 7 above)
Comment (ouuan) [reply content/en/_index.html]: Fine.

File: content/en/_index.html (hunk 6 above)
Comment (TFKls) [reply content/en/_index.html]: Alright then. :)

File: content/en/_index.html (hunk 2 above)
Comment (TFKls) [reply content/en/_index.html]: I did probably misunderstand this, as I thought the rephrasing was referring to the "If you have however chosen..." part. (Which I still do not think is necessary in that case).

Comment (ouuan) [reply content/en/_index.html]: > as I thought the rephrasing was referring to the "If you have however chosen..." part.

OK, I didn't notice that.

**Comment by ouuan** [comment]
@allcontributors add @TFKls as a contributor for content.
